internal/pkg/aws/cloudwatchlogs/cloudwatchlogs.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package cloudwatchlogs contains utility functions for Cloudwatch Logs client.
+package cloudwatchlogs
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+)
+
+const (
+	// SleepDuration is the sleep time for making the next request for log events.
+	SleepDuration        = 1 * time.Second
+	orderByLastEventTime = "LastEventTime"
+)
+
+var (
+	fatalCodes   = []string{"FATA", "FATAL", "fatal", "ERR", "ERROR", "error"}
+	warningCodes = []string{"WARN", "warn", "WARNING", "warning"}
+)
+
+type cloudwatchlogsClient interface {
+	DescribeLogStreams(input *cloudwatchlogs.DescribeLogStreamsInput) (*cloudwatchlogs.DescribeLogStreamsOutput, error)
+	GetLogEvents(input *cloudwatchlogs.GetLogEventsInput) (*cloudwatchlogs.GetLogEventsOutput, error)
+}
+
+// Service wraps an AWS Cloudwatch Logs client.
+type Service struct {
+	cwlogs cloudwatchlogsClient
+}
+
+// New returns a Service configured against the input session.
+func New(s *session.Session) *Service {
+	return &Service{
+		cwlogs: cloudwatchlogs.New(s),
+	}
+}
+
+// logStreams returns all name of the log streams in a log group.
+func (s *Service) logStreams(logGroupName string) ([]*string, error) {
+	resp, err := s.cwlogs.DescribeLogStreams(&cloudwatchlogs.DescribeLogStreamsInput{
+		LogGroupName: aws.String(logGroupName),
+		Descending:   aws.Bool(true),
+		OrderBy:      aws.String(orderByLastEventTime),
+	})
```

Comment:
It will get the logs from the tasks that are stopped. That's a good point! But the way we determine it is alive or not requires a lot more work to do. So I'd prefer to adding `--include-stopped` flag along with the flag allowing them to show the logs for specific task in the next PR, after we finish `app status`. What do you think?

Reply:
:shipit: sgtm!